infra/gcp/bash/lib.sh

```diff
@@ -389,34 +392,49 @@ function empower_group_to_admin_artifact_auditor() {
         --role="roles/iam.serviceAccountUser"
 }
 
+# Grant full privileges to the file promoter bot
+# $1: The GCP project
+# $2: The GCS region (optional)
+function empower_file_promoter() {
```

Comment:
Nothing is calling this, is that intentional?

Reply:
Fixed.
Now being called as part of `ensure_prod_gcs_bucket()`.